### PR TITLE
Sensors integration

### PR DESCRIPTION
**Pull request by magnuscruz**

### CODE REVIEW

**Comment by hmatalonga** [comment]
Hello Magnus, thank you for your PR. I take a look at it and get back to you once I reviewed it. I will also migrate the CI integration to github actions.

File: composer.json [hunk 1]
```diff
@@ -18,7 +19,9 @@
         "sentry/sentry-laravel": "^1.0",
         "spatie/laravel-backup": "^5.11",
         "spatie/laravel-cors": "^1.3",
-        "spatie/laravel-permission": "^2.28"
+        "spatie/laravel-permission": "^2.28",
+        "theseer/tokenizer": "^1.2",
```
Comment (hmatalonga) [inline composer.json:23]: These were dependencies from some composer update? They were auto included?

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    environment:
+      MYSQL_ROOT_PASSWORD: homestead
+      MYSQL_DATABASE: homestead
+      MYSQL_USER: homestead
+      MYSQL_PASSWORD: secret
+    ports:
+      - "3307:3306"
+    restart: always
+    volumes:
+      - "./create_tables.sql:/docker-entrypoint-initdb.d/1.sql"
```
Comment (hmatalonga) [inline docker-compose.yaml]: I think we don't need to include this docker-compose file. It won't be necessary for the production environment. Did you use this for local development right?

**Review by hmatalonga — COMMENTED** [review]
Overall it looks great. Thank you for taking the time to include these changes. I just left some comments to go over some details but it can be merged right after. Thanks :+1:

File: composer.json (hunk 1 above)
Comment (magnuscruz) [reply composer.json:23]: Yes, it did.

**Comment by hmatalonga** [comment]
Hi @magnuscruz, I have updated the CI workflow. No longer using travis CI in favour of Github actions.
I have made a couple of changes. I have updated the composer lock file to be compatible with version 2. This will require you to update your branch and override the composer.lock with the new one. Then when the CI passes it is safe to build and deploy.

**Comment by hmatalonga** [comment]
After you test the new features locally and you need to deploy them to production, let me know. I will give you access to the server so that you can monitor the database and manage the services.

**Review by hmatalonga — APPROVED** [review]

**Pull request merged**